README.md

```diff
@@ -34,6 +34,16 @@ testing towards [tianocore](https://www.tianocore.org/) utilizing
 * binutils (using `objcopy`)
 * Go >= 1.16
 
+## Available sbctl Packages
```

Comment:
```suggestion
## Available sbctl packages
```

Reply:
I am used to https://en.wikipedia.org/wiki/Title_case.

As this is not used in the rest of the README I am fine with this.